src/Honeybee_ExportEPC.py

```diff
@@ -149,11 +192,138 @@ def main(HBZones):
                               constr + "; facing " + \
                               directionDict[orientation] + ","+ " 0 "
                     else:
-                        if srfType == 1 or srfType == 2:
-                            print constr + ","+  \
-                              str(srfData[srfType][constr][orientation]["area"])
-                        else:
-                            print constr + \
-                              directionDict[orientation] + ","+ \
-                              str(srfData[srfType][constr][orientation]["area"])
+                        if srfType == 1:
+                            roofarea = int(srfData[srfType][constr][orientation]["area"])
+                            roofA.append(roofarea)
+                        if srfType == 2:
+                            floorarea = int(srfData[srfType][constr][orientation]["area"])
+                            floorA.append(floorarea)
+                        if srfType == 5:
+                            glazingarea = int(srfData[srfType][constr][orientation]["area"])
+                            glazingA.append(glazingarea)
+                            glazingorientation = directionDict[orientation]
+                            glazingO.append(glazingorientation)
+                            if glazingorientation == "N":
+                                GlazAreaNorth = GlazAreaNorth + glazingarea
+                            if glazingorientation == "NW":
+                                GlazAreaNorthWest = GlazAreaNorthWest + glazingarea
+                            if glazingorientation == "W":
+                                GlazAreaWest = GlazAreaWest + glazingarea
+                            if glazingorientation == "SW":
+                                GlazAreaSouthWest = GlazAreaSouthWest + glazingarea
+                            if glazingorientation == "S":
+                                GlazAreaSouth = GlazAreaSouth + glazingarea
+                            if glazingorientation == "SE":
+                                GlazAreaSouthEast = GlazAreaSouthEast + glazingarea
+                            if glazingorientation == "E":
+                                GlazAreaEast = GlazAreaEast + glazingarea
+                            if glazingorientation == "NE":
+                                GlazAreaNorthEast = GlazAreaNorthEast + glazingarea
+                            #print glazingO
+                        if srfType == 0:
```

Comment:
I agree with @chriswmackey's comment that even if you want to have this outside main function I suggest to wrap them inside a function to avoid duplication in your code.